sig-instrumentation/charter.md

```diff
@@ -0,0 +1,55 @@
+# SIG Instrumentation Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Owns best practices for cluster observability through metrics, logging, and
+events across all Kubernetes components and development of relevant components.
+
+### In scope
+
+Coordinating metric requirements of different SIGs for other components through finding common APIs.
+
+#### Code, Binaries and Services
+
+[List of subprojects](https://github.com/kubernetes/community/tree/master/sig-instrumentation#subprojects)
+
+- Components required for any Kubernetes cluster in regards to observability. ([kubernetes-incubator/metrics-server](https://github.com/kubernetes-incubator/metrics-server), [kubenernetes/heapster](https://github.com/kubernetes/heapster))
+- Interfaces/API definitions required for any Kubernetes cluster in regards to observability. ([kubernetes/metrics](https://github.com/kubernetes/metrics), [kubernetes-incubator/custom-metrics-apiserver](https://github.com/kubernetes-incubator/custom-metrics-apiserver))
+- Well established but optional components or adapters for Kubernetes clusters, if endorsed by members. Each component must have two or more members as maintainers. ([kubernetes/kube-state-metrics](https://github.com/kubernetes/kube-state-metrics), not yet officially owned by SIG Instrumentation, but an example prospect for this category: [DirectXMan12/k8s-prometheus-adapter](https://github.com/DirectXMan12/k8s-prometheus-adapter))
+
+#### Cross-cutting and Externally Facing Processes
+
+- Guidance for instrumentation in order to ensure consistent and high quality instrumentation of core Kubernetes components. This includes:
+  - Reviewing any instrumentation related changes and additions.
+  - Guidance on what should be instrumented as well as dimensions of the same.
+  - Creating, adding and maintaining the Kubernetes instrumentation guidelines.
+  - Coordinate cross SIG instrumentation efforts.
+
+### Out of scope
+
+- Processing of signals. For example ingesting metrics, logs, events into external systems.
+- Dictating what states must result in an alert. Suggestions or opt-in alerts may be in scope.
+
```

Comment:
i would say alert management is outside the scope entirely?

Reply:
we've been discussing have a base set of 'suggested alerts' around core kubernetes functionality.  things like 99% API response time, pod startup time and then make it very clear that the actual thresholds need to be tuned by the user but provide some sane defaults.  would that be too prescriptive?

Reply:
Yes, what @coffeepac said is correct. We have been discussing that suggestions for alerts could be something we can own, as that it already falls into most members daily work, and this could allow to centralize suggested alert sets more and better.